app/code/Magento/Persistent/Helper/Data.php

```diff
@@ -49,18 +49,26 @@ class Data extends \Magento\Framework\App\Helper\AbstractHelper
      */
     protected $_modulesReader;
 
+    /**
+     * @var \Magento\Store\Model\StoreManagerInterface
+     */
+    protected $storeManager;
```

Comment:
I think it is bad idea to use `protected` for new properties.
Let's use `private` here.

```suggestion
    private $storeManager;
```